screens/Navigation/SplitScreenNavigation/SplitRightStackNavigation.tsx

```diff
@@ -109,9 +103,8 @@ export default function SplitRightStackNavigation({
           {WebviewPreviewNav()}
           {ProfileNav()}
           {GroupNav()}
-          {GroupLinkNav()}
```

Comment:
Group link nav was not needed, `GroupInvite` replaced by `JoinGroup` to be more explicit